test/check-application

```diff
@@ -294,6 +307,12 @@ class TestApplication(testlib.MachineCase):
         hasTTY = m.execute("podman inspect --format '{{.Config.Tty}}' busybox-with-tty")
         self.assertEqual(hasTTY, 'true')
 
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+        b.wait_present('#containers-containers tr:contains("0.0.0.0:6000->5000/tcp, 0.0.0.0:6001->5001/udp")')
+        ports = m.execute("podman inspect --format '{{.NetworkSettings.Ports}}' busybox-with-tty");
+        self.assertIn('6000 5000 tcp', ports)
+        self.assertIn('6001 5001 udp', ports)
```

Comment:
Can you please also cover removal of port redirections? All the way up to having none, so that we keep covering that case as well?